upi/vsphere/nsx_network/variables.tf

```diff
@@ -0,0 +1,77 @@
+variable "nsx_manager" {
```

Comment:
If I read the patch correctly these variables are also defined in upi/vsphere/variables.tf - is that intentional?

Reply:
removed. thanks